src/modules/objects/objects.types.ts

```diff
@@ -0,0 +1,235 @@
+import { IPagination } from '@studiohyperdrive/pagination';
+
+import { MULTI_MATCH_QUERY_MAPPING, QueryType } from './elasticsearch/consts';
+
+import { ContactInfo } from '~shared/types/types';
+
+export interface QueryBuilderConfig {
+	AGGS_PROPERTIES: Array<SearchFilterField>;
+	MAX_COUNT_SEARCH_RESULTS: number;
+	MAX_NUMBER_SEARCH_RESULTS: number;
+	NEEDS_FILTER_SUFFIX: { [prop in SearchFilterField]?: string };
+	NUMBER_OF_FILTER_OPTIONS: number;
+	READABLE_TO_ELASTIC_FILTER_NAMES: { [prop in SearchFilterField]?: string };
+	DEFAULT_QUERY_TYPE: { [prop in SearchFilterField]?: QueryType };
+	OCCURRENCE_TYPE: { [prop in Operator]?: string };
+	VALUE_OPERATORS: Array<Operator>;
+	ORDER_MAPPINGS: { [prop in OrderProperty]: string };
+	MULTI_MATCH_FIELDS: Array<SearchFilterField>;
+	MULTI_MATCH_QUERY_MAPPING: typeof MULTI_MATCH_QUERY_MAPPING;
+	NEEDS_AGG_SUFFIX: { [prop in SearchFilterField]?: string };
+}
+
+export enum MediaFormat {
+	VIDEO = 'video',
+	AUDIO = 'audio',
+}
+
+export interface ObjectFile {
+	name: string;
+	alternateName: string;
+	description: string;
+	schemaIdentifier: string;
+	ebucoreMediaType: string;
+	ebucoreIsMediaFragmentOf: string;
+	embedUrl: string;
+}
+
+export interface ObjectRepresentation {
+	name: string;
+	alternateName: string;
+	description: string;
+	schemaIdentifier: string;
+	dctermsFormat: string;
+	transcript: string;
+	dateCreated: string;
+	files: ObjectFile[];
+}
+
+export interface ObjectMedia {
```

Comment:
Let's use IeObject if Object throws an error
then it is closer to the database naming: objects.ie
ie => Intellectual Entity

Reply:
Adjusted